src/components/UserLogin/index.js

```diff
@@ -30,6 +31,13 @@ const UserLogin = ({ gssoUrl, submitText }) => {
         <p className="lbh-body">
           Please contact your manager if you have issues signing in.
         </p>
+
+        <p className="lbh-body text-on-mobile-view">
```

Comment:
I would name this class based on something to do with its content, like `support-text`or `support-link`.  That way, if the design changes to show this on eg tablet as well as mobile, then we don't have to remember to update it to keep the readability intact.